sympy/holonomic/holonomic.py

```diff
@@ -520,7 +520,7 @@ def _singularics_to_ord(self):
             y0 = []
             for i in range(a):
                 y0.append(S(0))
-            y0 += b
+            y0 += [j * factorial(a + i) for i, j in enumerate(b)]
```

Comment:
I was thinking of the case where  a = 1.0, for example. That might happen, in principle, when the exact roots of the indicial equation cannot be found.